test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/Images/TSNativeImageSource.cs

```diff
@@ -47,28 +47,6 @@ public void NativeImageSourceConstructor()
             tlog.Debug(tag, $"NativeImageSourceConstructor END (OK)");
         }
 
-        [Test]
-        [Category("P1")]
-        [Description("NativeImageSource Url.")]
-        [Property("SPEC", "Tizen.NUI.NativeImageSource.Url A")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "PRW")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void NativeImageSourceUrl()
-        {
-            tlog.Debug(tag, $"NativeImageSourceUrl START");
-
-            var testingTarget = new NativeImageSource(100, 50, NativeImageSource.ColorDepth.Bits16);
-            Assert.IsNotNull(testingTarget, "Can't create success object NativeImageSource");
-            Assert.IsInstanceOf<NativeImageSource>(testingTarget, "Should be an instance of NativeImageSource type.");
-
-            var result = testingTarget.Url;
```

Comment:
Could you just change it as `GenerateUrl();`? It will fix everything

Reply:
Done